src/alda/sound/midi.clj

```diff
@@ -199,3 +215,11 @@
       .getChannels
       (pmap stop-channel!)
       doall)))
+
+(defn play-sequence!
+  "Plays a sequence on a java midi sequencer."
+  [sequence]
+  (doto (MidiSystem/getSequencer)
```

Comment:
I noticed that we're creating and opening 2 Sequencer instances -- one here, and one in `score-to-sequence`. I'm curious if it would be better to have only a single Sequencer that we use to both create the sequence and play the sequence.

I imagine there is some overhead to calling `.open()` on a Sequencer -- anecdotally, there is a noticeable pause when you call `.open()` on a Synthesizer. So, if that is the case, there are things that we can do to avoid sitting around waiting for the Sequencer to open.